Aquality.Selenium/src/Aquality.Selenium/Browsers/BrowserFactory.cs

```diff
@@ -10,5 +13,21 @@ protected BrowserFactory()
         }
 
         public abstract Browser Browser { get; }
+
+        protected ArgumentOutOfRangeException LoggedWrongBrowserNameException
+        {
+            get
+            {
+                var message = AqualityServices.Get<ILocalizationManager>().GetLocalizedMessage("loc.browser.name.wrong");
+                var exception = new ArgumentOutOfRangeException(message);
+                AqualityServices.Logger.Fatal(message, exception);
```

Comment:
Is it possible to use injection via constructor here and don't use AqualityServices?

Reply:
reworked